src/registry_schemas/schemas/notice_of_withdrawal.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/notice_of_withdrawal",
+    "type": "object",
+    "definitions": {},
+    "title": "Notice of Withdrawal Filing",
+    "required": [
+        "noticeOfWithdrawal"
+    ],
+    "properties": {
+        "noticeOfWithdrawal": {
+            "type": "object",
+            "required": [
+                "filingId"
+            ],
+            "properties": {
+                "filingId": {
+                    "type": "string",
```

Comment:
filingId can only be numbers, so, maybe we can use "integer" for its type. reference: https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/filing.json#L99C12-L104C15